### PR TITLE
Create new page

### PR DESCRIPTION
**Pull request by andriyParashchuk**
Staging - https://jetthoughts-pr-432.surge.sh/developers

https://www.loom.com/share/e37d86e4a4c34e7daeae5e34aad4e255

### CODE REVIEW

File: _assets/css/layout/_dev-description.scss [hunk 1]
```diff
@@ -1,10 +1,8 @@
 .dev-description {
   min-height: auto;
   color: #fff;
-  background: #6fcfc1;
-  background: -moz-linear-gradient(45deg, #6fcfc1 0%, #4b9ce2 100%);
-  background: -webkit-linear-gradient(45deg, #6fcfc1 0%,#4b9ce2 100%);
-  background: linear-gradient(45deg, #6fcfc1 0%,#4b9ce2 100%);
+  background: #6fcfc1 url(https://images.unsplash.com/photo-1519389950473-47ba0277781c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80) 50% 50% no-repeat;
```
Comment (pftg) [inline _assets/css/layout/_dev-description.scss]: do not forget move this image to our assets

**Comment by pftg** [comment]
Deployed to https://jetthoughts-pr-432.surge.sh

**Review by ad-sys — APPROVED** [review]
lgtm

**Pull request merged**